src/github.rs

```diff
@@ -779,6 +779,24 @@ impl Issue {
         Ok(())
     }
 
+    pub async fn merge(&self, client: &GithubClient) -> anyhow::Result<()> {
+        let merge_url = format!("{}/pulls/{}/merge", self.repository().url(), self.number);
+
+        #[derive(serde::Serialize)]
+        struct MergeIssue<'a> {
+            commit_title: &'a str,
+        }
```

Comment:
As we continue working on this part, we could consider adding options for the merge method, we can send `merge_method` parameter here, perhaps something to be configurable in the toml by each repo. (Or check with the Rust folks for what do they want here)
(same for whatever we title the commits)

Reply:
Yeah definitely! I actually thought from where we can read those values ​​to change the default but it's something we'd have to ask the Rust team about.

Will comment that so we don't forget in future 😄